### PR TITLE
EODP3-43 Profile Edit

### PR DESCRIPTION
**Pull request by tamilau**

### CODE REVIEW

File: src/classes/DependentPicklists.cls [hunk 1]
```diff
@@ -0,0 +1,189 @@
+/**
```
Comment (P0und-cake) [inline src/classes/DependentPicklists.cls]: @tamilau Was there a test class for this in the SACSCOC repo as well?

**Comment by P0und-cake** [comment]
This looks good. Nice work! Here are some more general things I think we should look into before moving on:

What should we do with the edit button for the official data tab? Are we going to edit official data one day? Should it open the edit profile modal? Should we hide it all together?

When you click the profile photo in the nav and go to My Profile, can we fix the initial loading modal so it covers the entire page? Bonus points if we can make the opaque mask the this color: color__contentMask

File: src/classes/ContentDocumentUtil.cls [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/**
+ * Utility class for working with content documents. Created by Presence PG on 3/28/2019.
+ */
+
+public class ContentDocumentUtil {
```
Comment (P0und-cake) [inline src/classes/ContentDocumentUtil.cls]: @tamilau Can we have a unit test class for this?

File: src/classes/DependentPicklists.cls (hunk 1 above)
Comment (tamilau) [reply src/classes/DependentPicklists.cls]: There was not, just a test class for the other one.

File: src/classes/ContentDocumentUtil.cls (hunk 2 above)
Comment (tamilau) [reply src/classes/ContentDocumentUtil.cls]: Done

**Comment by tamilau** [comment]
@P0und-cake fyi, I pushed some other updates and fixes onto this as well so I didn't create too many branches for tiny fixes.

**Pull request merged**